pageserver/client/src/mgmt_api.rs

```diff
@@ -549,4 +553,58 @@ impl Client {
             }),
         }
     }
+
+    pub async fn ingest_aux_files(
+        &self,
+        tenant_shard_id: TenantShardId,
+        timeline_id: TimelineId,
+        aux_files: HashMap<String, String>,
+    ) -> Result<bool> {
+        let uri = format!(
+            "{}/v1/tenant/{}/timeline/{}/ingest_aux_files",
+            self.mgmt_api_endpoint, tenant_shard_id, timeline_id
+        );
+        let resp = self
+            .request_noerror(Method::POST, &uri, IngestAuxFilesRequest { aux_files })
+            .await?;
+        match resp.status() {
+            StatusCode::ACCEPTED | StatusCode::OK => Ok(true),
+            StatusCode::NOT_MODIFIED => Ok(false),
```

Comment:
Can this even happen?

Reply:
removed the branch and changed accepted -> ok